Ghidra/Features/Decompiler/src/decompile/cpp/printlanguage.hh

```diff
@@ -431,6 +431,22 @@ public:
   /// \param nm is the configuration description
   virtual void setCommentStyle(const string &nm)=0;
 
+<<<<<<< HEAD
```

Comment:
There's a bunch of diff lines that were left in.

Reply:
Did GitHub change their ui? Or is this just odd cause I'm on mobile, though 
 'desktop mode'.  Should probably revert the merge entirely, and handle any debase without pulling in all the commits into this or (which I'm guessing is where the conflict came from, but the website looks different/off)

Reply:
Yeah, this merge has not gone well! Thanks for spotting this, just sorting it out now